Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Amazon.Lambda.TestTool.BlazorTester60-pack.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
This is basically the same as the [.NET 5](https://github.com/aws/aws-lambda-dotnet/blob/master/Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Amazon.Lambda.TestTool.BlazorTester50-pack.csproj) version of the file minus the framework number changes. This file is used in our pipeline to ensure we get a separate package for each version of .NET.